<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Texture-based Terrain Synthesis</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QLabel" name="exem_widget">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255)</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="target_widget">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255)</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="result_widget">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255)</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>655</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_Target"/>
    <addaction name="actionLoad_Exemplar"/>
    <addaction name="actionSave_Output"/>
    <addaction name="actionSave_Exemplar"/>
   </widget>
   <widget class="QMenu" name="menuRun">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionRun_Synthesis"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuRun"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad_Target">
   <property name="text">
    <string>Load Target</string>
   </property>
  </action>
  <action name="actionLoad_Exemplar">
   <property name="text">
    <string>Load Exemplar</string>
   </property>
  </action>
  <action name="actionSave_Output">
   <property name="text">
    <string>Save Output</string>
   </property>
  </action>
  <action name="actionRun_Synthesis">
   <property name="text">
    <string>Run terrain synthesis</string>
   </property>
  </action>
  <action name="actionSave_Exemplar">
   <property name="text">
    <string>Save Exemplar</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
